src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int COLORS_LENGTH = Color.values().length;
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(COLORS_LENGTH);
+        return Color.values()[index].toString().toLowerCase();
```

Comment:
```suggestion
        return Color.values()[index].name().toLowerCase();
```
[Use name() for getting String representation of enum constants](https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants)